public/css/style.css

```diff
@@ -84,6 +84,89 @@
    max-width: 25%;
 }
 
+#modal1{
+  width: 55%;
+  height: 60%;
+}
+
+.modal-content{
+  height: 100%;
+}
+
+#form {
+  margin-top: 5%;
+}
+
+#fb, #twit, #google, #git {
+  width:38%;
+  padding: 0 0.7rem !important;
+  margin-left: 5%;
+  margin-bottom: 15%;
+  margin-top: 5%;
+}
+
+#fb {
+  background-color: #3b5998 !important;
```

Comment:
Try to minimize (if possible eliminate) your use of !important.
https://developer.mozilla.org/en/docs/Web/CSS/Specificity
![selection_012](https://cloud.githubusercontent.com/assets/17332992/20176473/3e262648-a748-11e6-986e-8d776e0c2a85.png)